### DISCUSSION TITLE
Excessive Header Size

### DISCUSSION THREAD

**Discussion by DanielHerr** [mdn/mdn-community#680] [Platform] [answered]
The header is taking up almost a third of the page for me. It would be nice for it to be redesigned to take up less space.

![Screenshot 2024-05-12 12 53 34 AM](https://github.com/mdn/mdn-community/assets/7622517/d2ba50d3-0dfd-4bf1-be10-c825fcffa1bc)
![Screenshot 2024-04-30 3 49 33 PM](https://github.com/mdn/mdn-community/assets/7622517/e84fd76d-3520-48be-8a93-a744325dda9e)

**Comment by bsmth** [comment]
Hi @DanielHerr thanks for posting. Do you want to share some information about your environment so it can be reproduced? I think it might be better to track in a Yari bug so the device & OS info is captured: 

https://github.com/mdn/yari/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml

Thanks a lot!

**Reply by DanielHerr** [reply, marked as answer]
Opened https://github.com/mdn/yari/issues/11121

**Comment by bsmth** [comment]
Thanks a lot for raising the issue in Yari. I'm going to close the discussion so we can track the bug there.
